homeassistant/components/binary_sensor/tod.py

```diff
@@ -0,0 +1,234 @@
+"""
+Support for showing the binary sensors represending current time of the day.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/binary_sensor.tod/
+
+"""
+from datetime import datetime, timedelta
+import logging
+import pytz
```

Comment:
Please keep the imports grouped. `isort` can help here if you don't want to do it manually.

Reply:
OK Done.